generators/readme/package.json

```diff
@@ -26,7 +26,7 @@
     "lodash": "4.17.21",
     "rcfile": "1.0.3",
     "react-docgen": "5.4.3",
-    "remark-gfm": "1.0.0",
+    "remark-gfm-v1": "npm:remark-gfm@1.0.0",
```

Comment:
`remark-gfm` is used by this workspace and the storybook workspace but unfortunately in different versions. I tried several approaches to have multiple versions of the same package installed in the same mono-repo, but found this to be the least intrusive solution.